app/frontend/components/common/alert.js

```diff
@@ -0,0 +1,12 @@
+const {h} = require('preact')
+
+const Alert = ({children, alertType = 'success', bodyClass = ''}) =>
```

Comment:
What's the reason for the `bodyClass`?

Reply:
It's a leftover prop that should've been removed, will remove